service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceFactory.java

```diff
@@ -60,6 +62,7 @@ private Microservice createMicroserviceFromDefinition(Configuration configuratio
     microservice.setPaths(ConfigurePropertyUtils.getMicroservicePaths(configuration));
     Map<String, String> propertiesMap = MicroservicePropertiesLoader.INSTANCE.loadProperties(configuration);
     microservice.setProperties(propertiesMap);
+    microservice.setEnvironment(configuration.getString(SERVICECOMB_ENV, DEFAULT_SERVICECOMB_ENV));
```

Comment:
PR title said that "instance" stage, but IMPL is microservice
so what's the real purpose?

Reply:
The real purpose is microservice,the title in issue is too old, it's been modified now.

Reply:
![image](https://user-images.githubusercontent.com/30716999/37441594-2c1b1068-283d-11e8-85c8-5adbade1b082.png)

Reply:
![image](https://user-images.githubusercontent.com/30716999/37441602-377a004a-283d-11e8-8406-ef38fc13714b.png)

Reply:
ok